.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
   - jdk: openjdk11
 
 script:
-  - sbt "+ test"
+  - sbt ";scalafmtSbtCheck; +scalafmtCheckAll; +test"
```

Comment:
Is the `+` really necessary here?

Reply:
You mean on the `scalafmtCheckAll`? It's not strictly necessary since there's no version-specific code at the moment, but if we do add it (for Dotty or whatever), it would be really easy to forget to change this.

Reply:
Fair point!